python/mxnet/image/image.py

```diff
@@ -85,6 +84,46 @@ def imread(filename, *args, **kwargs):
     return _internal._cvimread(filename, *args, **kwargs)
 
 
+def imresize(src, w, h, interp):
+    r"""Resize image with OpenCV.
+
+    Note: `imread` uses OpenCV (not the CV2 Python library).
+    MXNet must have been built with USE_OPENCV=1 for `imresize` to work.
+
+    Parameters
+    ----------
+    src : NDArray
+        source image
+    w : int, required
+        Width of resized image.
+    h : int, required
+        Height of resized image.
+    interp : int, optional, default='1'
+        Interpolation method (default=cv2.INTER_LINEAR).
```

Comment:
is it possible to add all the possible values here, so the user doesn't have to go to another page to figure this out